frontend/pages/model/[modelId]/access-request/[accessRequestId]/review.tsx

```diff
@@ -0,0 +1,142 @@
+import { ArrowBack } from '@mui/icons-material'
+import { Button, Card, Container, Divider, Grid, Paper, Stack, Typography } from '@mui/material'
+import { useGetAccessRequest, useGetAccessRequestsForModelId } from 'actions/accessRequest'
+import { useGetModel } from 'actions/model'
+import { postReviewResponse, useGetReviewRequestsForModel } from 'actions/review'
+import { useRouter } from 'next/router'
+import { useState } from 'react'
+import Loading from 'src/common/Loading'
+import ReviewWithComment, { ResponseTypeKeys } from 'src/common/ReviewWithComment'
+import UserDisplay from 'src/common/UserDisplay'
+import MultipleErrorWrapper from 'src/errors/MultipleErrorWrapper'
+import Link from 'src/Link'
+import MessageAlert from 'src/MessageAlert'
+import Wrapper from 'src/Wrapper'
+import { mutate } from 'swr'
+import { formatDateString } from 'utils/dateUtils'
+import { getErrorMessage } from 'utils/fetcher'
+
+export default function AccessRequestReview() {
+  const router = useRouter()
+  const { modelId, accessRequestId }: { modelId?: string; accessRequestId?: string } = router.query
+
+  const [errorMessage, setErrorMessage] = useState('')
+
+  const { model, isModelLoading, isModelError } = useGetModel(modelId)
+  const { accessRequest, isAccessRequestLoading, isAccessRequestError } = useGetAccessRequest(modelId, accessRequestId)
+  const { mutateAccessRequests } = useGetAccessRequestsForModelId(modelId)
+  const { mutateReviews } = useGetReviewRequestsForModel({
+    modelId,
+    accessRequestId: accessRequestId as string,
+  })
+
+  async function handleSubmit(decision: ResponseTypeKeys, comment: string, role: string) {
+    setErrorMessage('')
+    if (!modelId || !accessRequestId) {
+      return setErrorMessage('Could not find model ID')
+    }
+
+    const res = await postReviewResponse({
+      modelId,
+      role,
+      comment,
+      decision,
+      accessRequestId,
+    })
+
+    if (!res.ok) {
+      setErrorMessage(await getErrorMessage(res))
+    } else {
+      mutate(
```

Comment:
Is this needed if we're calling `mutateReviews` below? If it is needed, could you explain why?

Reply:
I'm not sure it is needed, it's left over from the previous review code. Removing it seems to work fine.